### ISSUE TITLE
JIRA Server should required or warn about missing protocol

### ISSUE THREAD

**Issue by DyegoCosta** [square/jirafy#17]
Right now if we do not specify the protocol of the JIRA Server url it will fail to resolve the server, building an url like `https://github.com/<user>/<repo>/compare/<JIRA Server>/browse/SR-1810?expand=1`

It could use a default `https` or at least warn the user that a protocol is required

**Comment by sshaw** [comment]
I agree here. There seems to be some quirks with this extension and I think this would make things slightly easier.

**Comment by yannickoo** [comment]
I have added a pull request for this which adds validation for the JIRA Server field 🎉
